### ISSUE TITLE
Define minimal node version

### ISSUE THREAD

**Issue by cre8** [openwallet-foundation-labs/sd-jwt-js#110]
We should define the minimal node version in the package json.

Additional goal:
In this step, we should also consider to run the tests as matrix to validate that it is running for multiple node versions. Doing so we need to check which step should upload the codecov report since the test run in parallel (or check if we can split it up and merge the different coverage reports laster). Right now vitest is doing browser and node test in sequence and is merging the report in one final file.

**Comment by lukasjhan** [comment]
Are you going to write in package.json that it can be used on node 16 or higher?

**Comment by cre8** [comment]
I can open a PR for this

**Comment by cre8** [comment]
Done
